plugins/woocommerce/includes/class-wc-countries.php

```diff
@@ -1433,11 +1433,17 @@ public function get_country_locale() {
 					),
 					'VN' => array(
 						'state'     => array(
-							'required' => false,
-							'hidden'   => true,
+							'priorty'  => 70,
```

Comment:
Incorrect/typo-ish array key. Also, let's use `43`, then the `state` field can be right after `country` field (its current priority is 40 - [src](https://github.com/woocommerce/woocommerce/blob/018ce13b49bfa8a0f7653b5c2bd4c11dbaffd322/plugins/woocommerce/includes/class-wc-countries.php#L711)). 


```php
							'priority' => 43,
```